starwars/src/components/Cards.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Cards = props => {
+  console.log(props.cardsProp);
+  return(
```

Comment:
good job using your console logs to check on what it is you have on props and state in your app. practice with the react dev tools and ger familiar with them as we will be using them a lot from now on